plugins/core/resources/js/core/services/feedback.service.es

```diff
@@ -1,5 +1,12 @@
-angular.module('core').service('feedback', function($log, ajentiVersion, ajentiPlatform, ajentiPlatformUnmapped) {
-    this.enabled = true; // TODO
+angular.module('core').service('feedback', function($log, ajentiVersion, ajentiPlatform, ajentiPlatformUnmapped, customization) {
+    if (customization.plugins.core.enableMixpanel === false)
```

Comment:
```suggestion
    this.enabled = customization.plugins.core.enableMixpanel
```

Reply:
Thanks, but I think there's a problem when the var is not defined : it will be false per default.
It was my first commit yesterday, and then I got back to enable mixpanel per default, when the var is not defined.

Reply:
you could then write it as `this.enabled = customization.plugins.core.enableMixpanel !== false`

Reply:
Yes, I love it, it's pretty much better, I've done a second commit for this, thank you !